cert_manager.tf

```diff
@@ -33,6 +33,7 @@ module "cert_manager_sa" {
 }
 
 resource "helm_release" "cert_manager" {
+  count           = var.enable_cert_manager ? 1 : 0
```

Comment:
Should we also specify this conditional install for the above `module "cert_manager_sa"`?

Reply:
@maxsxu we will always want the SA to be created, but in the new provisioning scenario Terraform will no longer be managing components via Helm.